### ISSUE TITLE
E3 Error con Test case 4 de BasicHybridCards

### ISSUE THREAD

**Issue by EnzoMorata** [RodrigoToroIcarte/IIC2113-2023-1#213]
Hola, estoy teniendo un error en el test n°4 de 06-BasicHybridCards. En particular, tengo una duda del output esperado. En la línea 469 del test, STONE COLD STEVE AUSTIN juega [MANEUVER] *Collar & Elbow Lockup*, pero no ejecuta el 'tratar' de aplicar daño por haberla jugado como maniobra. Ya que tengo entendido, que únicamente las cartas híbridas no aplican daño al jugarlas como acción.

**Comment by RodrigoToroIcarte** [comment]
Lo que pasa es que `Collar & Elbow Lockup` hace `1D`. Pero el oponente es `MANKIND`. Su habilidad resta 1 a todo el daño que recibe. Por eso es que `Collar & Elbow Lockup` no le hace daño.

**Comment by EnzoMorata** [comment]
Claro, al menos mi código hasta el momento imprimir que se le realizaba 0 de daño. Entonces, siempre que se aplique 0 de daño puedo asumir que no de debiese imprimir nada?

**Comment by RodrigoToroIcarte** [comment]
Sí. Los test son consistentes en eso. Si el daño es cero, no se muestra nada :+1:

**Comment by EnzoMorata** [comment]
Ya, muchas gracias!
